tests/hardware_tests/_backupreachable_xy_theta.py

```diff
@@ -0,0 +1,363 @@
+import json
+
+import cv2
+import magnum as mn
+import numpy as np
+import quaternion
+import rospy
+from cv_bridge import CvBridge
+from geometry_msgs.msg import Quaternion, Transform, TransformStamped, Vector3
+from message_filters import ApproximateTimeSynchronizer, Subscriber
+from scipy import stats as st
+from sensor_msgs.msg import Image
+from spot_rl.models.yolov8predictor import YOLOV8Predictor
+from spot_rl.utils.utils import construct_config
+from spot_rl.utils.utils import ros_topics as rt
+from spot_wrapper.spot import Spot
+from std_msgs.msg import String
+
+
+def get_3d_point(cam_intrinsics, pixel_uv, z):
+    # Get camera intrinsics
+    fx = float(cam_intrinsics["focal_length"]["x"])
+    fy = float(cam_intrinsics["focal_length"]["y"])
+    cx = float(cam_intrinsics["principal_point"]["x"])
+    cy = float(cam_intrinsics["principal_point"]["y"])
+
+    # print(fx, fy, cx, cy)
+    # Get 3D point
+    x = (pixel_uv[0] - cx) * z / fx
+    y = (pixel_uv[1] - cy) * z / fy
+    return np.array([x, y, z])
+
```

Comment:
...
What is the use of this test?
Is it more like experiment than a test?

Files in `test/hardware_tests/` are for automated CI testing.